releasenotes/2.1.0.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.0.0
+date: 2016-11-15 12:00:00
+release_notes: true
+permalink: /releasenotes/2.0.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.0 represents a significant milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](https://s.apache.org/NbYK). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
```

Comment:
2.1

Reply:
Also, if you put a placeholder for the link such as `https://s.apache.org/jclouds210` you can then generate a link for the announcement using that name and you won't have to edit the post later.